charts/telepresence/templates/agent-configmap.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if not .Values.rbac.only }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: telepresence-agents
+  namespace:  {{ include "telepresence.namespace" . }}
+data:
+  agentInjector: |
```

Comment:
This was an alternative solution to get the agent aware of the managed namespaces. It can be removed completely since we now pass them in the `TELEPRESENCE_MANAGED_NAMESPACES` env.